### PR TITLE
bspwm colors

### PR DESCRIPTION
**Pull request by R0flcopt3r**
Created a colorscheme for bspwm. It replicates the colorscheme used for i3wm.

### CODE REVIEW

File: bspwm/colors [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+cyan="#0AAEB3"
+brightcyan="#53FDE9"
+
+white="#D0BFA1"
+brightwhite="#FCE8C3"
+
+orange="#D75F00"
+brightorange="#FF8700"
+
+xgrey1="#262626"
```
Comment (MindTooth) [inline bspwm/colors:28]: xgrey -> xgray (US edition)

Comment (R0flcopt3r) [reply bspwm/colors]: fixed.

**Comment by roosta** [comment]
Thank you for your contribution, don't use bspwm myself, but this looks inline with the i3 theme so I'm merging.

**Comment by roosta** [comment]
I think the color variables like bright[color] should be named bright_[color] in line with other themes.

**Comment by R0flcopt3r** [comment]
That's very strange, I was sure I used the same source for this PR and #7... I'll sort that out after the weekend.

**Comment by MindTooth** [comment]
Screengrabs?

File: bspwm/srcery [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+brightorange="#FF8700"
+
+xgray1="#262626"
+xgray2="#303030"
+xgray3="#3A3A3A"
+xgray4="#444444"
+xgray5="#4E4E4E"
+
+bspc config normal_border_color "$xgray1"
+bspc config active_border_color "$xgrey3"
```
Comment (MindTooth) [inline bspwm/srcery:35]:
```suggestion
bspc config active_border_color "$xgray3"
```

**Pull request merged**